src/Components/DonationFormModal.js

```diff
@@ -67,6 +180,7 @@ const DonationFormModal = () => {
                   ownerStory: e.target.value
                 })
               }
+              required
```

Comment:
Good catch making this required so we don't have to contend with making things more difficult for the backend.